Monika After Story/game/zz_reactions.rpy

```diff
@@ -2787,3 +2787,72 @@ label mas_reaction_gift_clothes_mocca_bun_blackandwhitestripedpullover:
     if giftname is not None:
         $ store.mas_filereacts.delete_file(giftname)
     return
+
+init 5 python:
+    # TODO: Add a way to generalize this
+    if not renpy.seen_label("mas_reaction_gift_noudeck"):
+        addReaction("mas_reaction_gift_noudeck", "noudeck", is_good=True)
+
+label mas_reaction_gift_noudeck:
+    python:
+        mas_giftCapGainAff(0.5)
+        # She keeps the deck at any aff
+        mas_unlockGame("nou")
+        mas_unlockEVL("monika_explain_nou_rules", "EVE")
+
+    if mas_isMoniNormal(higher=True):
+        m 1wub "Oh!{w=0.3} A deck of cards!"
+        m 3eua "And I think I know how to play this game!"
+        m 1esc "I heard it might {i}affect{/i} your relationships with the people you're playing with."
+
+        if mas_isMoniAff(higher=True):
+            show monika 5eubsa at t11 zorder MAS_MONIKA_Z with dissolve_monika
+            m 5eubsa "But I know our relationship can stand much more than just a simple card game~"
+            m 5hubsa "Ehehe~"
+            show monika 1eua at t11 zorder MAS_MONIKA_Z with dissolve_monika
+
+        else:
+            m 1hub "Ahaha!"
+            m 1eua "I'm just kidding, [player]."
+
+        m 1eua "Have you ever played 'NOU', [player]?{nw}"
+        $ _history_list.pop()
+        menu:
+            m "Have you ever played 'NOU', [player]?{fast}"
```

Comment:
I'm kinda thinking it's in our best interest to ref the OG game here, otherwise this question doesn't work out (as nobody would have played "NOU" before this point)

Reply:
True, but then the dialogue wouldn't really work. She'd ask about "Uno," but explain how to play "NOU." Any ideas?

Reply:
```
    m “Have you ever played Uno, [player]?”
    m “This game is very similar!”
    m “Now let me tell you the rules of NOU…”
```

and so on?

Reply:
John added a second question where Monika asks about "Uno"